examples/keras_recipes/creating_tfrecords.py

```diff
@@ -0,0 +1,290 @@
+"""
+Title: Creating TFRecords
+Author: [Dimitre Oliveira](https://www.linkedin.com/in/dimitre-oliveira-7a1a0113a/)
+Date created: 2021/02/25
+Last modified: 2021/02/25
+Description: Converting data to TFRecords files.
```

Comment:
Suggestion:

TFRecord (without the "s") (https://www.tensorflow.org/tutorials/load_data/tfrecord?hl=fr-EN)

```suggestion
Description: Converting data to the TFRecord format.
```